stable/datacube-wps/templates/deployment.yaml

```diff
@@ -66,16 +66,16 @@ spec:
               secretKeyRef:
                 name: {{ .Values.database.existingSecret }}
                 key: postgres-password
-          - name: AWS_ACCESS_KEY_ID
-            valueFrom:
-              secretKeyRef:
-                name: {{ .Values.aws.existingSecret }}
-                key: aws_access_key_id
-          - name: AWS_SECRET_ACCESS_KEY
-            valueFrom:
-              secretKeyRef:
-                name: {{ .Values.aws.existingSecret }}
-                key: aws_secret_access_key
+          # - name: AWS_ACCESS_KEY_ID
+          #   valueFrom:
+          #     secretKeyRef:
+          #       name: {{ .Values.aws.existingSecret }}
+          #       key: aws_access_key_id
+          # - name: AWS_SECRET_ACCESS_KEY
+          #   valueFrom:
+          #     secretKeyRef:
+          #       name: {{ .Values.aws.existingSecret }}
+          #       key: aws_secret_access_key
```

Comment:
If `wps` do not need AWS credentials, then why cannot it be removed from the deployment template file?

Reply:
Will likely to be reused in the future version as many issues are uncertain at this stage.